### PR TITLE
expose xml.sax Exceptions

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

**Comment by gvanrossum** [comment]
Please investigate the CI failure. It says
```
stdlib/2.7/xml/sax/__init__.pyi:1: error: Cannot find module named '_exceptions'
```
Is this unique to Python 2.7? If not, can you update the Python 3 stubs too?

**Comment by gvanrossum** [comment]
(Most likely you shouldn't import from _exceptions but just define the exceptions in the public stub.)

**Comment by mr-c** [comment]
Thanks, I have this fixed locally. The stubs are the same for Python 2.7 and 3.x

Shall I make a `stdlib/2and3/xml/sax` directory and put them there?

**Comment by gvanrossum** [comment]
I'd prefer not to move files around -- just copy the additions to all
places that currently have xml stubs.

We can make it a separate refactor to move the xml stubs to 2and3 once mypy
actually supports sys.version_info checking. (If you care about that you
can add an issue to the typeshed tracker.)

On Thu, Jun 9, 2016 at 9:50 AM, Michael R. Crusoe <notifications@github.com>
wrote:

> Thanks, I have this fixed locally. The stubs are the same for Python 2.7
> and 3.x
>
> Shall I make a stdlib/2and3/xml/sax directory and put them there?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/python/typeshed/pull/280#issuecomment-224956599>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/ACwrMrTwVaob-My5HSb2-JlQQSS944TTks5qKERDgaJpZM4IxxdK>
> .
>



-- 
--Guido van Rossum (python.org/~guido)

**Comment by mr-c** [comment]
Sure thing; looks ready to me.

**Comment by gvanrossum** [comment]
The PY2 stuff looks good.

For PY3, can you test explicitly that a program that uses both xml.sax and xml.etree can be checked with --python-version set to 3.2, 3.3, 3.4, 3.5? The multiple package versions may cause some of these to fail.

File: stdlib/2.7/xml/sax/__init__.pyi [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Stubs for xml.sax (Python 2.7)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+class SAXException(Exception):
+    def __init__(self, msg, exception=None): ...
```
Comment (gvanrossum) [inline stdlib/2.7/xml/sax/__init__.pyi]: Oh wait, can't you manually add types to these signatures, insofar as these methods are documented?

**Comment by gvanrossum** [comment]
I'll just merge this, we'll see if anybody complains.

**Pull request merged**